src/styles/countries.css

```diff
@@ -1,3 +1,36 @@
+.country_card_container {
+  background-color: rgb(165, 14, 99);
+  transition: all 0.3s;
+  margin-bottom: 0.5rem;
+  margin-top: 3rem;
+  min-width: 39rem;
+}
+
+.country_card_container:hover {
+  transition: all 0.4s;
+  transform: scale(1.15);
+}
+
+.grid {
+  display: flex;
+  flex-wrap: wrap;
+  width: 70%;
+  justify-content: space-between;
+  margin: 10px auto;
+  gap: 1rem;
+  text-align: center;
+}
```

Comment:
- [ ] Please know that while you tried to fix what the previous reviewer mentioned something else changed, the countries are no longer split into 2 columns, this is how it looks:

  ![image](https://user-images.githubusercontent.com/20567503/207126253-8b80ae10-3bb5-42c1-81d4-b09faeb22b47.png)

  And this is how it should look:

  ![image](https://user-images.githubusercontent.com/20567503/207126711-8d1f3f02-1e83-4197-ac56-b5b95196a929.png)

  The countries are split into two columns and the colors are alternating between light pink and dark pink as the previous reviewer mentioned, kindly fix this issue as it is a critical one since you should follow the design provided correctly!
  
  Hint: if you use `grid` instead of `flex` in your display here it will help you fix the issue.